src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -263,3 +263,10 @@ const post911Periods = createSelector(
 );
 
 export const servedAfter911 = formData => !!post911Periods(formData).length;
+
+export const needsToEnter781 = formData =>
+  _.get('view:selectablePtsdTypes.view:combatPtsdType', formData, false) ||
+  _.get('view:selectablePtsdTypes.view:noncombatPtsdType', formData, false);
```

Comment:
Okay, so the 781 isn't for `Military Sexual Trauma` or `Personal Assault`, is that right?

Reply:
Correct, and users will be able to fill out both.